update_id.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+
+export PROJ_FILE="./Loop.xcodeproj/project.pbxproj"
+
+echo -n "Enter a new handle.  A single word, usually lower-case, no special characters: "
+read NEW_HANDLE
+
+sed -i '' "s/MAIN_APP_BUNDLE_IDENTIFIER = com\..*\.Loop/MAIN_APP_BUNDLE_IDENTIFIER = com.${NEW_HANDLE}.Loop/g" "${PROJ_FILE}"
```

Comment:
From previous experience, it's not a good idea to `sed` the pbxproj file. It can be a bit of a wild west. However, I can move this variable out into a `xcconfig` file (which I should have done a while ago) to make this replace simpler.